app/src/main/java/org/oppia/app/splash/SplashActivity.kt

```diff
@@ -0,0 +1,27 @@
+package org.oppia.app.splash
+
+import android.os.Bundle
+import android.view.WindowManager
+import androidx.appcompat.app.AppCompatActivity
+import org.oppia.app.R
+import android.content.res.Configuration
+
+/*
+* This activity acts as a container to SplashFragment
+* There is a background theme set to this activity to display app logo until the app is initialized
+*/
+class SplashActivity : AppCompatActivity() {
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
```

Comment:
Although we don't yet have a style guide outlined in the overall architecture document, let's follow a hybrid of JetBrain's Kotlin style guide and Google's Java style guide. In this case, per https://google.github.io/styleguide/javaguide.html#s4.2-block-indentation let's use 2 spaces for indentation and 4 spaces for continuation (which I think I configured at the project level for Kotlin--maybe make sure that you're using the project configuration for Kotlin in your IDE and try reformatting the code via the IDE).

We can evaluate later whether this is the spacing that we want, though I'm inclined to try and fit more code on a line rather than increasing whitespace since it makes it harder to read on smaller screens.

Reply:
Fixed indentation